yarn.lock

```diff
@@ -12487,7 +12487,7 @@ react-native-dynamic@^1.0.0:
 
 "react-native-fs@https://github.com/oceanbit-dev/react-native-fs.git#all-commands":
   version "2.16.4"
-  resolved "https://github.com/oceanbit-dev/react-native-fs.git#11f131f0e90f100f94b3e1daa86cf7a3479b973c"
+  resolved "https://github.com/oceanbit-dev/react-native-fs.git#cd4784f414c5b0b07de108001ae359a1cdd64028"
```

Comment:
I believe I made some hacky workarounds in the `react-native-fs` forked repo to get this all working. I'd like to make sure that we're using the central branch for everything (Android + iOS changes) and that this doesn't break things in Android

Reply:
Just confirmed that this is the newest commit in:

https://github.com/oceanbit-dev/react-native-fs/tree/all-commands

This should be good-to-go with Android versions